spec.bs

```diff
@@ -10166,43 +10159,43 @@ The [=k-anonymity overrides=] are checked before and take precedence over the
     </table>
 </figure>
 
-The [=remote end steps=] are:
+<div algorithm="remote end steps">
+The <dfn export for="Protected Audience">remote end steps</dfn> are:
+
 1. If |parameters| is not a JSON [[ECMASCRIPT#sec-json-object|Object]], return a
-     [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+     [=error|WebDriver error=] with [=error code=] [=invalid argument=].
 1. Let |owner| be the result of [=getting a property=] named `"owner"` from
     |parameters|.
-1. If |owner| is [=undefined=] or is not a [=string=], return a
-   [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+1. If |owner| is {{undefined}} or is not a [=string=], return a
+   [=error|WebDriver error=] with [=error code=] [=invalid argument=].
 1. Let |owner origin| be the result running [=parse an https origin=] on |owner|.
-1. If |owner origin| is failure, return a [=WebDriver error=] with
-   [=WebDriver error code=] [=invalid argument=].
-
+1. If |owner origin| is failure, return a [=error|WebDriver error=] with
+   [=error code=] [=invalid argument=].
 1. Let |name| be the result of [=getting a property=] named `"name"` from
     |parameters|.
-1. If |name| is [=undefined=] or is not a [=string=], return a
-   [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
-
+1. If |name| is {{undefined}} or is not a [=string=], return a
+   [=error|WebDriver error=] with [=error code=] [=invalid argument=].
 1. Let |hashes| be the result of [=getting a property=] named `"hashes"` from
     |parameters|.
-1. If |hashes| is [=undefined=] or is not an [=list=], return a
-   [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+1. If |hashes| is {{undefined}} or is not a [=list=], return a
+   [=error|WebDriver error=] with [=error code=] [=invalid argument=].
 1. Let |override| be a [=map=] whose [=map/keys=] are [=SHA-256=] hashes and
    whose [=map/values=] are [=k-anonymity records=].
 1. [=list/For each=] |hash string| of |hashes|:
   1. If |hash string| is not a [=string=], return a
-     [=WebDriver error=] with [=WebDriver error code=] [=invalid argument=].
+     [=error|WebDriver error=] with [=error code=] [=invalid argument=].

```

Comment:
interesting, both assembly and webdriver spec defined [=error=], and this auto directs to [=error=] defined in webassemly, without conflicts warning.
Probably easist to revert this back to the `text: WebDriver error; url: dfn-error` thing you had.

Reply:
I changed the reference for the web assembly error, since it's not actually exported.